### PR TITLE
feedback links mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
This adds the links on the intro / relogin page to send feedback

### CODE REVIEW

File: shared/login/forms/index.native.js [hunk 1]
```diff
@@ -101,17 +101,34 @@ const Intro = (props: Props) => (
           Your device has been revoked, please log in again.
         </Text>
       </Box>}
-    <Box style={{...globalStyles.flexBoxColumn, flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+    <Box
```
Comment (chrisnojima) [inline shared/login/forms/index.native.js]: this isn't feel bleed otherwise, weirdly

File: shared/login/forms/index.native.js [hunk 2]
```diff
@@ -101,17 +101,34 @@ const Intro = (props: Props) => (
...
+        justifyContent: 'center',
+        height: '100%',
+        width: '100%',
+      }}
+    >
       <Icon type="icon-keybase-logo-80" />
       <Text style={stylesHeader} type="HeaderBig">Join Keybase</Text>
       <Text style={stylesHeaderSub} type="Body">Folders for anyone in the world.</Text>
       <Button style={stylesSignupButton} type="Primary" onClick={props.onSignup} label="Create an account" />
+      <Box style={{flex: 1, backgroundColor: 'red'}} />
```
Comment (chrisnojima) [inline shared/login/forms/index.native.js]: just put a spacer here so we can have nice spacing no matter the size

File: shared/login/forms/index.native.js [hunk 3]
```diff
@@ -101,17 +101,34 @@ const Intro = (props: Props) => (
...
       <Text style={stylesLoginHeader} type="Body" onClick={props.onLogin}>Already on Keybase?</Text>
       <Button style={stylesLoginButton} type="Secondary" onClick={props.onLogin} label="Log in" />
+      <Text style={stylesFeedback} type="BodySmallPrimaryLink" onClick={props.onFeedback}>
```
Comment (chrisnojima) [inline shared/login/forms/index.native.js]: the new link

File: shared/login/forms/intro.js [hunk 4]
```diff
@@ -12,37 +11,38 @@ import {compose, branch, renderComponent} from 'recompose'
 
 import type {TypedState} from '../../constants/reducer'
 
+const mapStateToProps = (state: TypedState) => ({
```
Comment (chrisnojima) [inline shared/login/forms/intro.js]: just hoist this into helpers like our new code

File: shared/login/login/container.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as Creators from '../../actions/login/creators'
```
Comment (chrisnojima) [inline shared/login/login/container.js]: this is really moving index.js to container like our new code. moving things into helpers and using recompose to eliminate the helper class

Comment (chrisnojima) [inline shared/login/login/container.js]: https://github.com/keybase/client/pull/7321/files#diff-e9defe33f89336e53d7c9dd6c6f6a898L1

File: shared/login/login/index.native.js [hunk 6]
```diff
@@ -0,0 +1,100 @@
+// @flow
```
Comment (chrisnojima) [inline shared/login/login/index.native.js]: this is really a rename of index.render.native like our new code (https://github.com/keybase/client/pull/7321/files#diff-b7d36e9e1a3349fd6ce4289bc1e10f59L1)

File: shared/login/login/index.native.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+        label: 'Show typing',
+        checked: this.props.showTyping,
+        onCheck: check => {
+          this.props.showTypingChange(check)
+        },
+      },
+    ]
+
+    return (
+      <NativeScrollView>
```
Comment (chrisnojima) [inline shared/login/login/index.native.js]: I added a scrollview so it all fits with keyboard up

File: shared/login/login/index.native.js [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+            </Text>
+          </UserCard>
+          <Text style={{marginTop: globalMargins.xlarge}} type="BodyBigLink" onClick={this.props.onSignup}>
+            Create an account
+          </Text>
+          <Text
+            style={{margin: globalMargins.small, marginTop: globalMargins.large, alignSelf: 'flex-end'}}
+            type="BodySmallPrimaryLink"
+            onClick={this.props.onFeedback}
+          >
```
Comment (chrisnojima) [inline shared/login/login/index.native.js]: the new link

File: shared/login/login/routes.js [hunk 9]
```diff
@@ -11,8 +11,13 @@ import SetPublicName from '../register/set-public-name'
 import Success from '../register/success'
 import RegisterError from '../register/error'
 import GPGSign from '../register/gpg-sign'
+import Feedback from '../../settings/feedback-container'
 
 const recursiveLazyRoutes = I.Seq({
+  feedback: {
```
Comment (chrisnojima) [inline shared/login/login/routes.js]: we can get to feedback directly

File: shared/login/routes.js [hunk 10]
```diff
@@ -3,10 +3,15 @@ import {RouteDefNode} from '../route-tree'
 import loginRoutes from './login/routes'
 import signupRoutes from './signup/routes'
 import Intro from './forms/intro'
+import Feedback from '../settings/feedback-container'
 
 const routeTree = new RouteDefNode({
   component: Intro,
   children: {
+    feedback: {
```
Comment (chrisnojima) [inline shared/login/routes.js]: we can get to feedback directly

File: shared/settings/feedback-container.desktop.js [hunk 11]
```diff
@@ -0,0 +1,4 @@
+// @flow
+// Not implemented
+const FeedbackContainer = () => null
```
Comment (chrisnojima) [inline shared/settings/feedback-container.desktop.js]: the old feedback-container was just native only so lets just enforce that at the container level

File: shared/settings/nav/index.desktop.js [hunk 12]
```diff
@@ -44,6 +44,7 @@ function SettingsNav({badgeNumbers, selectedTab, onTabChange}: Props) {
         badgeNumber={badgeNumbers[deleteMeTab]}
         onClick={() => onTabChange(deleteMeTab)}
       />
+      <SettingsItem text="Sign out" selected={false} badgeNumber={0} onClick={onLogout} />
```
Comment (chrisnojima) [inline shared/settings/nav/index.desktop.js]: useful to test this on desktop and enough people have asked for this

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by keybase-ci-visdiff** [comment]
The commits b15ec2c48e02...a313a6feff61 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867637036-b15ec2c48e02-a313a6feff61-linux/Settings_Nav-Normal-b15ec2c48e02.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867637036-b15ec2c48e02-a313a6feff61-linux/Settings_Nav-Normal-a313a6feff61.png)  
   Settings_Nav-Normal comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits b15ec2c48e02...32d76edc010f introduce visual changes on linux. <details><summary>:mag_right: 3 changed</summary>

 * Changed: **Tracker-Platforms** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Tracker-Platforms-b15ec2c48e02.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Tracker-Platforms-32d76edc010f.png)  
   Tracker-Platforms comparison
 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Tracker-New_user_Scroll_1-b15ec2c48e02.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Tracker-New_user_Scroll_1-32d76edc010f.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **Settings_Nav-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Settings_Nav-Normal-b15ec2c48e02.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496867757883-b15ec2c48e02-32d76edc010f-linux/Settings_Nav-Normal-32d76edc010f.png)  
   Settings_Nav-Normal comparison
</summary>

**Pull request merged**